packages/flutter_tools/lib/src/test/watcher.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import '../base/io.dart' show Process;
+
+/// Callbacks for reporting progress while running tests.
+class TestWatcher {
```

Comment:
It could be, but technically it's okay to use the base class as a no-op implementation, so I'll leave it as is.